src/mokernels/moinput.jl

```diff
@@ -24,7 +26,7 @@ The first `out_dim` elements represent all outputs for the first input, the seco
 
 See [Inputs for Multiple Outputs](@ref) in the docs for more info.
 """
-struct MOInputIsotopicByFeatures{S,T<:AbstractVector{S}} <: AbstractVector{Tuple{S,Int}}
+struct MOInputIsotopicByFeatures{S,T<:AbstractVector{S}} <: AbstractMOInput{Tuple{S,Int}}
```

Comment:
What are the pros/cons of including the tuple construct in the abstract type parameterisation? Alternatively we could also have
```suggestion
struct MOInputIsotopicByFeatures{S,T<:AbstractVector{Tuple{S,Int}}} <: AbstractMOInput{S}
```